packages/components/src/components/DaCard.vue

```diff
@@ -127,6 +127,13 @@ export default {
   & .ls-blur-up-img.ls-inview.ls-original-loaded {
     opacity: 0;
   }
+
+  &:before {
+    position: absolute;
+    content: '';
+    height: 100%;
+    width: 100%;
+  }
```

Comment:
I think it fits better inside the `.animate-cards` selector.
If the cards animation is off there is no transition and this pseudo-element is not needed.
Anyway, this works great! 🥳

Reply:
Definitely true on that.